projects/dotcms-field-elements/src/components/dot-date-range/dot-date-range.e2e.ts

```diff
@@ -1,147 +1,349 @@
+                expect(input.getAttribute('disabled')).toBeDefined();
+            });
+
+            it('should not break with invalid data --> falsy', async () => {
+                element.setProperty('disabled', 0);
+                await page.waitForChanges();
+                expect(input.getAttribute('disabled')).toBeNull();
+            });
         });
 
-        describe('date range set', () => {
-            beforeEach(async () => {
-                const days = await page.findAll('.flatpickr-day');
-                days[5].click();
-                days[8].click();
+        describe('name', () => {
+            const value = 'test';
+
+            it('should render attribute in label and select', async () => {
+                element.setProperty('name', value);
+                await page.waitForChanges();
+                const idValue = input.getAttribute('id');
+                expect(idValue).toBe('dot-test');
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(labelElement.getAttribute('name')).toBe(value);
+            });
+
+            it('should not render attribute in label and select', async () => {
+                const idValue = input.getAttribute('id');
+                expect(idValue).toBe('dot-daterange');
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(labelElement.getAttribute('name')).toBe('daterange');
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [1, 2, '3'];
+                element.setProperty('name', wrongValue);
+                await page.waitForChanges();
+                const idValue = input.getAttribute('id');
+                expect(idValue).toBe('dot-123');
+            });
+        });
+
+        describe('label', () => {
+            it('should render attribute in label', async () => {
+                const value = 'test';
+                element.setProperty('label', value);
+                await page.waitForChanges();
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(labelElement.getAttribute('label')).toBe(value);
+            });
+
+            it('should not break with invalid data', async () => {
+                const wrongValue = [1, 2, '3'];
+                element.setProperty('label', wrongValue);
+                await page.waitForChanges();
+                const labelElement = await dotTestUtil.getDotLabel(page);
+                expect(labelElement.getAttribute('label')).toEqual('1,2,3');
+            });
+        });
+
+        describe('presetLabel', () => {
+            it('should render attribute in preset label', async () => {
+                const value = 'test';
+                element.setProperty('presetLabel', value);
+                await page.waitForChanges();
+                const presetLabel = await page.find('label');
```

Comment:
You're grabbing the `label` inside `dot-label`

```suggestion
                const presetLabel = await page.find('.dot-range__body label');
```

Reply:
leaves the preset dropdown empty